src/scss/constants.scss

```diff
@@ -304,6 +306,12 @@ $z-layers: (
     background-color: $black;
   }
 
+  &::-webkit-scrollbar-thumb:hover {
+    background-color: rgba($white, 0.5);
+  }
```

Comment:
Good catch 👍 . The color is also more commonly used as `opaque-white()` function

Reply:
fixed in that PR https://github.com/sparkletown/sparkle/pull/2387